packages/flutter_localizations/lib/src/l10n/cupertino_pt.arb

```diff
@@ -0,0 +1,22 @@
+{
+  "datePickerHourSemanticsLabelOne": "$hour hora",
+  "datePickerHourSemanticsLabelOther": "$hour horas",
+  "datePickerMinuteSemanticsLabelOne": "1 minuto",
+  "datePickerMinuteSemanticsLabelOther": "$minute minutos",
+  "datePickerDateOrder": "dmy",
+  "datePickerDateTimeOrder": "date_time_dayPeriod",
+  "anteMeridiemAbbreviation": "AM",
+  "postMeridiemAbbreviation": "PM",
+  "todayLabel": "Hoje",
+  "alertDialogLabel": "Alerta",
+  "timerPickerHourLabelOne": "hora",
+  "timerPickerHourLabelOther": "horas",
+  "timerPickerMinuteLabelOne": "min",
+  "timerPickerMinuteLabelOther": "min",
+  "timerPickerSecondLabelOne": "s",
+  "timerPickerSecondLabelOther": "s",
+  "cutButtonLabel": "Cortar",
+  "copyButtonLabel": "Copiar",
+  "pasteButtonLabel": "Colar",
+  "selectAllButtonLabel": "Selecionar Tudo"
```

Comment:
The selectAllButtonLabel should be exactly as the iOS SO because it overrides it. In this language the System uses Upper Case only in the first word.
```suggestion
  "selectAllButtonLabel": "Selecionar tudo"
```